WordPress/src/main/res/values/styles.xml

```diff
@@ -559,4 +559,13 @@
         <item name="android:textSize">@dimen/text_sz_medium</item>
         <item name="android:textStyle">bold</item>
     </style>
+
+    <style name="UploadProgressBar" parent="Widget.AppCompat.ProgressBar.Horizontal">
+        <item name="android:indeterminateOnly">false</item>
+        <item name="android:progressDrawable">@drawable/wp_progress_horizontal</item>
```

Comment:
I'm curious why we need this custom progress drawable, and what the purpose is of the minHeight, maxHeight and scaleY?